src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    static final String White_Space = " ";
+    static final int Date_Index = 0;
+    static final int Name_Index = 1;
+    static final int Hours_Index = 2;
+    static final int Salary_Index = 3;
```

Comment:
Constant names use CONSTANT_CASE: all uppercase letters, with each word separated from the next by a single underscore
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names